packages/rocketchat-mapview/server/settings.coffee

```diff
@@ -0,0 +1,3 @@
+Meteor.startup ->
+	RocketChat.settings.add 'MapView_Enabled', false, {type: 'boolean', group: 'Message', section: 'Google Maps', public: true, i18nLabel: 'MapView Enabled', i18nDescription: 'MapView_Enabled_Description'}
```

Comment:
Can you use the pattern `MapView_Enabled` for i18nLabel like you are using for description? And can you add these keys in `en.i18n.json` file? If you do that our translators/contributors will be able to see theses strings to translate in lingohub.